pkgs/applications/virtualization/rkt/default.nix

```diff
@@ -2,14 +2,14 @@
 , fetchurl, fetchFromGitHub }:
 
 let
-  coreosImageRelease = "794.1.0";
-  coreosImageSystemdVersion = "222";
+  coreosImageRelease = "835.9.0";
```

Comment:
We should keep this aligned to https://github.com/coreos/rkt/blob/v0.14.0/stage1/usr_from_coreos/coreos-common.mk#L14